Automattic/wp-calypso
# Mulstisite Scheduled Updates: Adapt global style layout

Closes https://github.com/Automattic/wp-calypso/issues/90151

## Proposed Changes

* Adapt multisite scheduled updates inside the new layout
* Update the `schedule-list` component to support compact version of presentation
* Update CSS to match the Figma proposal
* Update CSS to support mobile resolutions

## Testing Instructions

* Open live link
* Go to `/plugins/scheduled-updates?flags=plugins/multisite-scheduled-updates`
* Check how the feature acts inside the new layout
* Check how existing schedule list works

![Screen Capture on 2024-05-08 at 19-22-05](https://github.com/Automattic/wp-calypso/assets/1241413/06294292-922f-4bc5-894a-3cfdfe9b2ed1)

![Screen Capture on 2024-05-09 at 12-35-58](https://github.com/Automattic/wp-calypso/assets/1241413/9f150e27-3209-4982-94c3-5c80cf7582ba)


## Pre-merge Checklist
